src/Car.js

```diff
@@ -0,0 +1,29 @@
+import { Random } from '@woowacourse/mission-utils';
+
+class Car {
+  #name;
+
+  #distance;
+
+  constructor(name) {
+    this.#name = name;
+    this.#distance = 0;
+  }
+
+  #move() {
+    const randomNumber = Random.pickNumberInRange(0, 9);
+    if (randomNumber >= 4) {
+      this.#distance += 1;
+    }
+  }
+
+  driving() {
+    this.#move();
+
+    const name = this.#name;
+    const currentDistance = this.#distance;
+    return { name, distance: currentDistance };
+  }
+}
```

Comment:
move메소드와 driving 메소드의 추상화 수준이 맞지 않는 것 같아요!!
현재 move 메소드는 driving 메소드를 가져다 쓰며 결과를 뱉고 있는데 이는 절차 지향적인 느낌이 있습니다.
따라서  추상화 수준을 맞추기 위해 return 하는 부분을 다른 메소드로 두시고 이 둘을 driving에서 호출 하시면 좋을 것 같습니다.
여기서 더 나아가 보자면 driving 메소드를 외부에 두고 이둘을 호출하는 것이 가장 추상화 수준을 맞출 수 있는 방법이라고 생각해요.
```suggestion
  move() {
    const randomNumber = Random.pickNumberInRange(0, 9);
    if (randomNumber >= 4) {
      this.#distance += 1;
    }
  }

  getCarInfo() {
    return { name: this.#name, distance:this.#distance };
  }
}
```

Reply:
이번에 다른 분들의 코드를 읽어보며 추상화 수준이라는 개념을 처음 접하게 됐는데, 확실히 추상화 수준에 맞지 않는 코드는 여러 역할을 수행하는 것처럼 느껴지거나 흐름이 어색하게 느껴지네요.
사실 저렇게 driving 메서드가 저렇게 return하도록 한 건 getter 메서드를 최대한 피해보고자하기 위함이었는데, 결국 driving 메서드가 move 호출과 getter 역할을 모두 수행한 셈이 되었네요..ㅎㅎ 깊이 있는 분석 감사합니다!